src/ILCompiler.ReadyToRun/src/Compiler/DependencyAnalysis/ReadyToRun/SignatureBuilder.cs

```diff
@@ -484,9 +521,36 @@ public ModuleToken GetModuleTokenForType(EcmaType type)
             return _resolver.GetModuleTokenForType(type);
         }
 
+        public ModuleToken GetModuleTokenForMethod(MethodDesc method)
```

Comment:
It's probably worth splitting `SignatureContext` out into its own source file now it's a couple hundred lines long

Reply:
Yes, I agree, it's kept on expanding recently.

Reply:
Fixed in 2nd commit.